### PR TITLE
Update colorRegistry.ts

### LINKED ISSUE #8748: listInactiveSelectionBackground color was too faint on both light and dark theme to be noticeable
Light theme - current view
light-theme-current-color

Light theme - suggested color
light-theme-new-color

Dark theme - current view
dark-theme-current-color

Dark theme - suggested color
dark-theme-new-color

The background colors of files selected in the explorer are too faint to be noticeable while the focus is outside of the explorer (e.g. while you're in the editor).

### PR DESCRIPTION
**Pull request by eligoldner**
Changed listInactiveSelectionBackground  colors for light/dark theme so that they are visible when not selected.

This PR fixes #8748

### CODE REVIEW

**Comment by eligoldner** [comment]
These are the actual values which are being used, so I guess it's hitting
the fallback. Can you please direct me to where the theme colors are
supposed to be for these elements?

Thanks,

Eli

**Comment by anthonydresser** [comment]
All of our default themes are defined in the extensions, https://github.com/microsoft/azuredatastudio/tree/master/extensions/theme-defaults/themes

**Comment by eligoldner** [comment]
Ok, I'll take a peek.
Thanks!

**Comment by anthonydresser** [comment]
We also have some themes defined https://github.com/microsoft/azuredatastudio/tree/master/extensions/theme-carbon/themes so which one you change will depend on which themes you are using.

File: src/vs/platform/theme/common/colorRegistry.ts [hunk 1]
```diff
@@ -349,7 +349,7 @@ export const listFocusBackground = registerColor('list.focusBackground', { dark:
...
 export const listActiveSelectionForeground = registerColor('list.activeSelectionForeground', { dark: Color.white, light: Color.white, hc: null }, nls.localize('listActiveSelectionForeground', "List/Tree foreground color for the selected item when the list/tree is active. An active list/tree has keyboard focus, an inactive does not."));
-export const listInactiveSelectionBackground = registerColor('list.inactiveSelectionBackground', { dark: '#37373D', light: '#E4E6F1', hc: null }, nls.localize('listInactiveSelectionBackground', "List/Tree background color for the selected item when the list/tree is inactive. An active list/tree has keyboard focus, an inactive does not."));
+export const listInactiveSelectionBackground = registerColor('list.inactiveSelectionBackground', { dark: '#5B5B60', light: '#BBBDC6', hc: null }, nls.localize('listInactiveSelectionBackground', "List/Tree background color for the selected item when the list/tree is inactive. An active list/tree has keyboard focus, an inactive does not."));
```
Comment (kburtram) [inline src/vs/platform/theme/common/colorRegistry.ts]: all changes in the 'vs' directory need to be proceeded b a comment `// {{SQL CARBON EDIT}}` so we can easily track modifications to the VS Code forked source code.

**Comment by kburtram** [comment]
@eligoldner I'll close this stale PR since it has a couple requests that haven't been addressed and so isn't currently in a merge-able state.  Please reopen addressing comments if we still want to take this.

**Pull request closed without merging**